astropy/coordinates/builtin_frames/galactocentric.py

```diff
@@ -88,76 +98,95 @@ class galactocentric_frame_defaults(ScienceState):
     _references = None
     _value = None
 
+    _registry = {
+        "v4.0": {
+            "value": MappingProxyType(
```

Comment:
I had to look up that this "produces a read-only view" - nice! Maybe add a one-line comment above the registry?

Reply:
done.